packages/pagination/examples/basic.md

```diff
@@ -93,3 +95,181 @@ initialState = {
   </Row>
 </Grid>;
 ```
+
+### Cursor Pagination
+
+#### Previous & Next
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { getColor } = require('@zendeskgarden/react-theming/src');
+
+const StyledCircle = styled.div`
+  margin: 0 ${props => props.theme.space.base}px;
+  border-radius: 100%;
+  height: ${props => props.theme.space.base * 3}px;
+  width: ${props => props.theme.space.base * 3}px;
+`;
+
+const CircleStroke = styled(StyledCircle)`
+  border: ${props =>
+    `0.6px ${props.theme.borderStyles.solid} ${getColor('grey', 300, props.theme)}`};
+`;
+
+const CircleFill = styled(StyledCircle)`
+  background: ${props => getColor('blue', 600, props.theme)};
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: ${props => `${props.theme.space.base * 5}px 0`};
+`;
+
+const VisuallyHidden = styled.div`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+  word-wrap: normal;
+`;
```

Comment:
Prefer the more restrained styling provided by https://github.com/zendeskgarden/css-components/blob/cff98fbf952f118ac60413ce4d9a24351872777d/packages/utilities/src/visibility.css#L18-L27 or just replace below with `<div className="u-visibility-screenreader" aria-live="polite">`, as the utility classes are available in Styleguidist.

Reply:
Awesome, I went with the utility classes provided by Styleguidist. Thanks for the suggestion 👍.